src/transformers/modeling_gguf_pytorch_utils.py

```diff
@@ -72,7 +71,7 @@ def load_gguf_checkpoint(gguf_checkpoint_path, return_tensors=False):
             and only loads the metadata in memory.
     """
     try:
-        from gguf import GGUFReader
+        from gguf import GGUFReader, dequantize
     except (ImportError, ModuleNotFoundError):
```

Comment:
Let's change how we check that gguf is installed. Can you use is_gguf_available() function just like how we usually do in transformers ? Also since we are doing a breaking change, it would be nice to force the user to install gguf >= 10.0 from now. So we need to check that he user have the right version + raise an error if that's not the case.

Reply:
Done.